pallets/data-preservers/src/benchmarks.rs

```diff
@@ -36,6 +36,17 @@ use {
     tp_traits::{ParaId, StorageDeposit},
 };
 
+/// Trait describing factory function for para_id.
+pub trait ArgumentFactory<AcocuntId> {
+    /// Factory function reserving parachain
+    fn reserve_para_id(para_id: ParaId);
```

Comment:
Is this unused?

Reply:
Yes removed in https://github.com/moondance-labs/tanssi/pull/783/commits/73b28788d8abcc176e94c59120d83842b0086065